cpu/driver/utils/verify-x86-cpufreq-driver.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+FILE=$(readlink -f ${BASH_SOURCE})
+NAME=$(basename $FILE)
+CDIR=$(dirname $FILE)
+
+verify_intel_cpufreq_driver()
+{
+    lscpu | grep " hwp " &> /dev/null
+    if [ $? -eq 0 ]; then
+	result=$(sudo rdmsr 0x770 2> /dev/null)
+	[ $? -ne 0 ] && result=0
+	if [ $result -ne 1 ]; then
+	    echo "intel system has HWP, but it is not enabled"
```

Comment:
Should we skip the test here ?

Reply:
I thought this is a setup the test should be detecting but good question!

Reply:
Sure, skipped.